runtime/tools/chainSpecRawBuilder/src/customSpec.js

```diff
@@ -0,0 +1,76 @@
+const { balancesToRaw } = require("./balances-module");
+const { systemToRaw } = require("./system-module");
+
+/**
+ * Class for the customSpec genesis configuration
+ */
+class CustomSpec{
+    name = "";
+    id = "local_testnet";
+    chainType =  "Local";
+    bootNodes = [];
+    telemetryEndpoints = null;
+    protocolId = null;
+    properties = null;
+    consensusEngine = null;
+    genesis = {};
```

Comment:
I don't think that we need these properties. We simply need f.e a `GenesisBuilder.convertToRaw(json)` a function that does all the job